### ISSUE TITLE
Give it an option to disable the worlds list that allows IFill commands/functions

### ISSUE THREAD

**Issue by goodroach** [Pocketkid2/Fill#4]
Self Explanatory, I have a server with a special kind of build plot where it creates a world for every plot. I'd like it if there was an option in the configs with some kind of boolean saying "allowAllWorlds" where it would override the list and simply allow all worlds in the server to use IFill.

**Comment by TylerS1066** [comment]
https://github.com/Pocketkid2/Fill/blob/master/src/main/resources/config.yml

```yaml
# In what worlds should the wand be enabled?
#   If a world name is not on this list, it will not work there
#   You may also need quotes if your world name has spaces
worlds:
  - world
  - world_nether
  - world_the_end
```

**Comment by goodroach** [comment]
> https://github.com/Pocketkid2/Fill/blob/master/src/main/resources/config.yml
> 
> ```yaml
> # In what worlds should the wand be enabled?
> #   If a world name is not on this list, it will not work there
> #   You may also need quotes if your world name has spaces
> worlds:
>   - world
>   - world_nether
>   - world_the_end
> ```

Yeah like a configuration option in there to override it and just allow all the worlds.

**Comment by Pocketkid2** [comment]
This is actually a great idea, I'm sorry I didn't get to this issue sooner. I will have that done today.

**Comment by Pocketkid2** [comment]
Implemented in v0.4.0
